cfy_manager/components/cli/cli.py

```diff
@@ -55,11 +55,14 @@ def _configure():
     use_cmd = ['cfy', 'profiles', 'use', 'localhost',
                '--skip-credentials-validation']
 
+    ssl_enabled = 'on' if config[MANAGER][SECURITY]['ssl_enabled'] else 'off'
+
     # TODO: Don't skip validation after CFY-7689 is done
     set_cmd = [
         'cfy', 'profiles', 'set', '-u', username,
         '-p', password, '-t', 'default_tenant',
-        '-c', EXTERNAL_CERT_PATH, '--skip-credentials-validation'
+        '-c', EXTERNAL_CERT_PATH, '--ssl', ssl_enabled,
```

Comment:
if `off`, perhaps we shouldnt have the `-c-`

Reply:
Decided that it's harmless enough